study-builder/studies/pancan/subs.conf

```diff
@@ -16,6 +16,12 @@
     "facebook": "https://www.facebook.com/joincountmein"
   },
 
+  "timeout": {
+    "secs_1_week": 604800,
+    "secs_2_week": 1209600,
+    "secs_3_week": 1814400,
```

Comment:
I factored out the delay numbers here so we can more easily tweaks these (e.g. down to minutes) when playing with things in `dev` or local.

Reply:
> "secs_1_week": 604800,
What kind of delays these parameters define?

Reply:
These define how many seconds are in 1/2/3 weeks.